CONTRIBUTING.md

```diff
@@ -6,7 +6,10 @@ We really appreciate and value contributions to OpenZeppelin Contracts for Cairo
 
 Before starting development, please [create an issue](https://github.com/OpenZeppelin/cairo-contracts/issues/new/choose) to open the discussion, validate that the PR is wanted, and coordinate overall implementation details.
 
-Also, consider that snake case is used for Cairo development in general due to its strong Python bias, but in order to be compliant with the standards, EIP implementations must use camelCase. Therefore, EIP-based contracts will have their external functions written in camelCase while the rest of the codebase will be in snake_case.
+Also, consider that snake case is used for Cairo development in general due to its strong Python bias.
+This project's opinionated approach consists of using snake_case for all internal functions and camelCasing all exposed functions and parameters.
```

Comment:
camelCasing all exposed functions and parameters implies all parameters?, including the ones for internal functions? I think it could be a little confusing.

Reply:
Yes, great point!